EIPS/eip-1.md

```diff
@@ -265,11 +265,15 @@ The `description` field in the preamble:
 
 When referring to an EIP by number, it should be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number.
 
+### IETF RFCs
+
+When referring to an IETF RFC by number, it must be written in the non-hyphenated form `RFC X`, where `X` is the RFC's assigned number. Each RFC that is referenced in an EIP **MUST** be accompanied by a relative markdown link to the RFC (in the `assets/eip-1/RFCs` directory) the first time it is referenced, and **MAY** be accompanied by a link on subsequent references. If the RFC is not currently in that directory, a separate pull request must be submitted which only includes the RFC in that directory.
+
 ### RFC 2119
 
-EIPs are encouraged to follow [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
+EIPs are encouraged to follow [RFC 2119](../assets/eip-1/RFCs/rfc2119.txt) for terminology and to insert the following at the beginning of the Specification section:
```

Comment:
It looks like you're using a lowercase directory, but an uppercase link. Not sure if that's bad or not.

Reply:
I thought I renamed the directory, huh. I'll fix that.